config/core/resources/integrationsink.yaml

```diff
@@ -297,6 +297,39 @@ spec:
                           description: The duration (in seconds) that the received messages are
                             hidden from subsequent retrieve requests after being retrieved by
                             a ReceiveMessage request.
+                    sns:
+                      type: object
+                      properties:
+                        arn:
+                          type: string
+                          title: Topic Name
+                          description: The SNS topic name name or Amazon Resource Name (ARN).
+                        region:
+                          type: string
+                          title: AWS Region
+                          description: The AWS region to access.
+                        autoCreateTopic:
+                          type: boolean
+                          title: Autocreate Topic
+                          description: Setting the autocreation of the SNS topic.
+                          default: false
+                        sessionToken:
+                          type: string
+                          title: Session Token
+                          description: Amazon AWS Session Token used when the user needs to assume
+                            a IAM role.
```

Comment:
this is different from spec.auth.secret `aws.secret...` ?

Reply:
and if it's a "token" I presume we don't want it to be in plaintext on a custom object ?

Reply:
I have a separate PR to remove this all together: https://github.com/knative/eventing/pull/8369

Reply:
Ok, let's track it somewhere?